client/src/component/inputText/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TextField from '@material-ui/core/TextField';
+
+import useStyles from './style';
+
+function InputField({ variant, onChange, ...rest }) {
+  const classes = useStyles();
+  return (
+    <TextField
+      className={classes.input}
+      variant={variant}
+      type="text"
```

Comment:
these lines are not needed, as the variant is default to `outlined` in our general theme, and the type may be changed according to the form like `password, email, ...`